.buildkite/test_containers.sh

```diff
@@ -7,6 +7,12 @@ set -o pipefail
 set -o nounset
 set -x
 
+echo "--- Cleanup docker"
+echo "Warning: deleting all docker containers and deleting ~/.ddev/Test*"
```

Comment:
I don't think ~/.ddev/Test* gets deleted, if we're still expecting that to happen.

Reply:
Great point - It's a copy-from-original-script problem. This script doesn't ever create anything in ~/.ddev, I'll fix that.